### PR TITLE
restructure install docs

### PR DESCRIPTION
**Pull request by nomicode**

### CODE REVIEW

File: docs/install-run/special/windows.txt [hunk 1]
```diff
@@ -38,13 +40,13 @@ Allow the ports CrateDB uses through the Windows Firewall:
 Install CrateDB
 ===============
 
-:doc:`Download the CrateDB Tarball <tarball>`, expand it and move to a
+:ref:`Download the CrateDB Tarball <basic-install>`, expand it and move to a
```
Comment (seut) [inline docs/install-run/special/windows.txt]: shouldn't this refer to  <install_targz> instead?

**Review by seut — CHANGES_REQUESTED** [review]

File: docs/install-run/special/windows.txt (hunk 1 above)
Comment (nomicode) [reply docs/install-run/special/windows.txt]: @seut that is no longer a valid reference. there is now only one page that documents the tarball install method, and that is `basic-install` in the getting started docs

Comment (seut) [reply docs/install-run/special/windows.txt]: @nslater hm and whats about that? https://github.com/crate/getting-started/pull/17/files#diff-60a5e6172bb738429e778d057d301612R35

Comment (nomicode) [reply docs/install-run/special/windows.txt]: @seut you're right! I was confused! sorry! thanks! pushed a fixup

**Review by seut — APPROVED** [review]

**Pull request merged**